### PR TITLE
Fix logins

### LINKED ISSUE #21: deprecated openid
Tried to login to feeder.gina.alaska.edu today and got the dedicated OpenID 2.0 from Google notification message.

https://support.google.com/accounts/answer/6206245?p=openid&rd=1

https://www.evernote.com/l/AFwb0rQzsOVM5Ka3chu0V1N0mh-Wd1DV2y0B/image.png

### PR DESCRIPTION
**Pull request by smacfarlane**
Fixes #21

### CODE REVIEW

**Comment by teknofire** [comment]
I assume most of these commits are cherry picked from the other pull request?

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 class AdminController < ApplicationController
-  before_filter :require_admin_auth
+  before_action :require_admin!
```
Comment (teknofire) [inline app/controllers/admin_controller.rb]: I think this can just go away, should be handled by the ability stuff right?

Comment (smacfarlane) [inline app/controllers/admin_controller.rb]: In it's current state no.  I'll need to re-work the abilities file to handle controller scope.

Comment (teknofire) [inline app/controllers/admin_controller.rb]: Ah yes that makes sense

**Comment by teknofire** [comment]
👍

**Pull request merged**